### ISSUE TITLE
Speech position for NPCs should be at a fixed position near their character

### ISSUE THREAD

**Issue by anneomcl** [punto-/mystery2#2]
Right now the player can move around while talking to people, but the NPC dialogue position is at a fixed point on the screen, and does not update if the player moves around.

Either we freeze the player during dialogue (I think this is best), or we have the position update as the player moves around.

**Comment by punto-** [comment]
Yeah, I think it's best to not allow too much activity while an event from
an esc is running. Right now the vm is able to run more than one task at a
time, but there's also a method 'can_interact' (left over from Dog
Mendonca, where no i teraction was allowed while an esc was running, and
only one esc could run). We could call that before any interaction happens
to avoid any race condition between esc events..
